pytest/tests/mocknet/mirror.py

```diff
@@ -451,6 +468,14 @@ def run_env_cmd(args, traffic_generator, nodes):
     parser.add_argument('--start-height', type=int)
     parser.add_argument('--unique-id', type=str)
     parser.add_argument('--local-test', action='store_true')
+    parser.add_argument('--type',
```

Comment:
same comment as in the other PR, --type seems ambiguous, can we rename this to something more specific like --host-type or --hosts (and maybe change --filter to --filter-hosts)?